README.md

```diff
@@ -97,138 +97,147 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
- - Added support for launching Settings app directly.
- - Fixed issue causing DisplayDescription not showing for PowerToys Run PluginAdditionalOption.
- - Fixed issue causing FileLocksmith 'Show File Locksmith in' setting not showing correct value.
- - Fixed issue causing Awake on/off toggle in Settings flyout not to work when Settings Awake page is opened.
+ - Text Extractor 2.0, with a new overlay, table mode and more Quality of Life improvements. Thanks [@TheJoeFin](https://github.com/TheJoeFin)!
 
 ### Documentation
 
- - Added documentation for PowerToys Run third-party plugins. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Fixed broken links in keyboardmanagerui.md. Thanks [@shubhsardana29](https://github.com/shubhsardana29)!
- - Updated core team in COMMUNITY.md.
- - Fixed broken links in ui-architecture.md. Thanks [@SamB](https://github.com/SamB)!
- - Updated community.valuegenerator.md with Base64DecodeRequest description.
+ - SECURITY.md was updated from 0.0.2 to 0.0.9. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
+ - Improved the README and main development document for clarity and completeness. Thanks [@codeofdusk](https://github.com/codeofdusk) and [@aprilbbrockhoeft](https://github.com/aprilbbrockhoeft)!
 
 ### Development
 
- - Updated test packages and StyleCop. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Condense NuGet Restore into MSBuild Tasks. Thanks [@snickler](https://github.com/snickler)!
+ - Fixed PowerToys Run DateTime plugin tests that were failing depending on locale, so that they can be run correctly on all dev machines.
+ - Fixed PowerToys Run System plugin tests that were failing for certain network interfaces, so that they can be run correctly on all dev machines. Thanks [@snickler](https://github.com/snickler)!
+ - Fixed a markdown bug on the GitHub /helped command.
+ - Switched build pipelines to a new agent pool. Thanks [@DHowett](https://github.com/DHowett)!
+ - New .cs files created in Visual Studio get the header added automatically. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 
-#### What is being planned for version 0.74
+#### What is being planned for version 0.75
 
-For [v0.74][github-next-release-work], we'll work on the items below:
+For [v0.75][github-next-release-work], we'll work on the items below:
 
  - Language selection
- - Modernize and refresh the UX of PowerToys based on WPF
+ - New utility: Environment Variables Editor. Here's a Work in Progress preview:
+![Environment Variables Editor WIP](https://github.com/microsoft/PowerToys/assets/26118718/f99532a8-5aae-481b-a662-19a95f4aa03d)
+
+ - New Settings homepage. Here's a Work in Progress preview:
+![PowerToys Settings Dashboard WIP](https://github.com/microsoft/PowerToys/assets/26118718/938a5715-0a9b-4fe9-9e15-adfec92da694)
+
+ - Modernize and refresh the UX of PowerToys based on WPF. Here's Work in Progress previews for PowerToys Run and Color Picker:
```

Comment:
"PowerToys" should be "toys" or "modules" (or something like that) because writing it like this means the whole package.

Reply:
"PowerToys Run" is the name of the utility we are talking about here.

Reply:
...as is Color Picker

Reply:
@Jay-o-Way , I've rephrased to `Here's Work in Progress previews for the modules "PowerToys Run" and "Color Picker":`
What do you think? :)